cmd/kubeadm/app/cmd/reset.go

```diff
@@ -182,6 +182,15 @@ func (r *Reset) Run(out io.Writer, client clientset.Interface) error {
 	}
 	resetConfigDir(kubeadmconstants.KubernetesDir, r.certsDir)
 
+	// Output help text instructing user how to remove iptables rules
+	msg := `
+	[reset] The reset process does not reset or clean up iptrables rules.
```

Comment:
Nit: typo in iptables

Reply:
D'oh silly rogue 'r'! Thanks for read-proofing!